quick-start/src/test/java/com/marklogic/quickstart/integrationtests/EndToEndAPITest.java

```diff
@@ -0,0 +1,350 @@
+import io.restassured.response.Response;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public class EndToEndAPITest extends HubTestBase {
+
+	@LocalServerPort
+	int port;
+
+	private IntegrationUtils utils = new IntegrationUtils();
+	
+	// Project Initialization variables
+	boolean projectInitStatus;
+	JsonPath projectInitJson;
+	Response projectInitResponse;
+	String projectPath;
+	
+	// Login Process variables
+	String loginResponse;
+	
+	// Clear Database variables
+	Response clearDbResponse;
+	
+	// Database stats variables
+	JsonPath statsJson;
+	Response statsResponse;
+	
+	// Create Entities variables
+	JSONObject createEntityJsonObj;
+	JsonPath getEntitiesJson;
+	List<EntityModel> entitiesList;
+	Response createEntityResponse;
+	Response getEnitiesResponse;
+	String createEntityJsonBody;
+	String entityName;
+	String responseEntityTitle;
+	
+	// Flow process variables
+	JsonPath createInputFlowJson;
+	JsonPath createHarmonizeFlowJSON;
+	JsonNode mlcpOptions;
+	JsonNode runHmJsonObj;
+	Map<String, Object> runFlowOptions;
+	Response createInputFlowResponse;
+	Response runIpFlowResponse;
+	Response createHarmonizeFlowResponse;
+	Response runHmFlowResponse;
+	Response deleteFlowResponse;
+	String flowType;
+	String respFlowName;
+	String respEntityName;
+	String respFlowType;
+	String pluginDir;
+	String inputPath;
+	String basePath;
+	String runHmJsonBody;
+	
+	// Map Creation variables
+	JsonNode mappingDataJsonNode;
+	JsonNode editMapRequestNode;
+	Response getMappingResponse;
+	Response createMapResponse;
+	String mapName;
+	String mappingData;
+	
+	// Job Status variables
+	JsonNode jobResultsNode;
+	Response getJobsResponse;
+	
+	@BeforeAll
+	public void setUp() {
+		createProjectDir();
+	}
+
+	@AfterAll
+	public void tearDown() {
+		deleteProjectDir();
+	}
+
+	@Before
+	public void serverSetup() {
+		RestAssured.port = port;
+	}
+
+	@Test
+	public void modelMappingE2ETests() throws JSONException, IOException, InterruptedException {
+		
+		// Tests api's to initialize the project folder and set the environment configuration
+		projectInitResponse = utils.initilizeProjectConfiguration();
+		projectInitJson = projectInitResponse.jsonPath();
+		projectInitStatus =  projectInitJson.getBoolean("initialized");
+		projectPath =  projectInitJson.getString("path");
+		
+		assertEquals(200, projectInitResponse.statusCode());
+		assertTrue(projectInitStatus);
+		assertTrue(projectPath.contains("marklog-data-hub-develop/marklogic-data-hub/quick-start/ye-olde-project"));
+		System.out.println("projectInitResponse: "+ projectInitResponse.asString());
+		System.out.println("Project Init Test Passed");
+		
+		// Tests api's to login into the quick-start application. A session cookie is received in
+		// header which has to be attached to the subsequent requests for security
+		loginResponse = utils.doLogin();
+		
+		assertTrue(!loginResponse.isEmpty());
```

Comment:
assertFalse() with a positive assertion might be more readable?

Reply:
Changed it to assertFalse() for more readability.